cmd/snap/inhibit.go

```diff
@@ -126,6 +128,16 @@ func waitWhileInhibited(ctx context.Context, cli *client.Client, snapName string
 
 func getInfoAndApp(snapName, appName string, rev snap.Revision) (*snap.Info, *snap.AppInfo, error) {
 	info, err := getSnapInfo(snapName, rev)
+	if errors.As(err, &snap.NotFoundError{}) {
+		// Differentiate between snap not existing and missing current symlink.
```

Comment:
nitpick: I think I'd prefer this comment to be before the `if errors.As` line, since it refers to the outer error check on line 141. I initially read it as referring to the `dirErr != nil` vs `!exists || !isDir`, which confused me. But maybe that's just me :)

Reply:
Good point, Thanks! updated